editor/import/resource_importer_texture.cpp

```diff
@@ -86,79 +83,87 @@ void ResourceImporterTexture::_texture_reimport_normal(const Ref<CompressedTextu
 	singleton->make_flags[path].flags |= MAKE_NORMAL_FLAG;
 }
 
+inline void ResourceImporterTexture::_print_callback_message(const String &message) {
+#ifdef TOOLS_ENABLED
+	EditorToaster::get_singleton()->popup_str(message);
+#endif
+	print_line(message);
+}
+
 void ResourceImporterTexture::update_imports() {
 	if (EditorFileSystem::get_singleton()->is_scanning() || EditorFileSystem::get_singleton()->is_importing()) {
-		return; // do nothing for now
+		return; // Do nothing for now.
```

Comment:
```suggestion
		return; // Do nothing for now.
```

Maybe change this to a fix me mentioning what we want to do in the future or delete the comment.

Reply:
I think the comment means "when the filesystem is doing something else" by "now"

Reply:
That comment is pretty confusing, then.

Reply:
I've changed it to "Do nothing while the editor filesystem is doing something else". Still maybe a little confusing, though less than before.